cumulusci/tasks/bulkdata/load.py

```diff
@@ -82,13 +82,13 @@ def _run_task(self):
 
             self.logger.info(f"Running Job: {name}")
             result = self._load_mapping(mapping)
-            if result != "Completed":
+            if not result.startswith("Completed"):
```

Comment:
Here and below, I think it'd be easier to read the intent of the code if we checked for specific values (e.g. `if result not in ("Completed", "CompletedWithFailures"):`) instead of checking for a substring

Reply:
https://github.com/SFDO-Tooling/CumulusCI/pull/1446/commits/c723246f05e15c48f39a03d1c0ea221716ada8d6#diff-a9551a5c355a86c23df29e1d9686489cL96